package.json

```diff
@@ -34,10 +34,6 @@
 	],
 	"license": "AGPL-3.0-only",
 	"devDependencies": {
-		"@emotion/react": "^11.10.5",
```

Comment:
I think these dependencies should stay as they are needed by commons

Reply:
But why in the _devDependencies_? They're already defined in the _dependencies_ which I think is the correct section to declare them